### DISCUSSION TITLE
How to implement caching using docker internal functionality?

### DISCUSSION THREAD

**Discussion by beeyev** [mlocati/docker-php-extension-installer#690] [Q&A]
Here is an example, if I want to install some alpine packages with caching the sources to speed up the future build, I can do something like this:
```dockerfile
RUN --mount=type=cache,target=/var/cache/apk/ apk add --quiet --update --no-cache \
    bash \
    tzdata
```

How can I cache `mlocati/php-extension-installer` source dependencies the same way? If it's possible, of course.

**Comment by dignac3** [comment]
I get problem optimizing the cache for this script too
